.travis.yml

```diff
@@ -10,20 +10,20 @@ env:
   global:
   - DOCKER_REPO=$TRAVIS_REPO_SLUG
 before_install:
-- apt-cache madison docker-engine
-- travis_retry sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine
+- apt-cache madison docker-ce
```

Comment:
@michaelmoss : you're very welcome! Thank you for not only finding the problem, but root-causing and fixing it!

With regards to the "gitiot" statement (lol, I love that), I cannot point fingers!  I resisted learning about branching for my first 1.5 years of using git, and only became comfortable after using it thoroughly for 1.5 more years after that.